v4/source/files.yaml

```diff
@@ -5,29 +5,45 @@
       summary: Upload a file
       description: |
         Uploads a file that can later be attached to a post.
+
+        This request can either be a multipart/form-data request with a channel_id, files and optional
+        client_ids defined in the FormData, or it can be a request with the channel_id and filename
+        defined as query parameters with the contents of a single file in the body of the request.
```

Comment:
isn't this alternative request type suppose to be of content type `application/octet-stream` ?

Reply:
It can be any type. The server doesn't really care what it is as long as it's not multipart

Reply:
I'm 0/5 here, it's pretty common not to send a content-type when you're just sending bytes